<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>830</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>830</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QComboBox" name="com_box">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="connect_btn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QComboBox" name="baud_box">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>5</number>
     </property>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256000</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Servo Kiri</string>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>230</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="servo11_slider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="servo1_slider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>110</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="servo9_slider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>87</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="servo11_offset">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>230</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="servo7_slider">
     <property name="geometry">
      <rect>
       <x>89</x>
       <y>150</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>90</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="servo1_offset">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="cursorPosition">
      <number>1</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="servo3_offset">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="servo5_slider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>85</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo 9</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="servo7_offset">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo 11</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo 7</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="servo5_offset">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo 5</string>
     </property>
    </widget>
    <widget class="QSlider" name="servo3_slider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>105</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo 3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="servo9_offset">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Servo Kanan</string>
    </property>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo 6</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo 12</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo 2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="servo2_offset">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="servo12_offset">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>230</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="servo10_slider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>101</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="servo8_slider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>90</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="servo12_slider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="servo4_slider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>85</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="servo2_slider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>78</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="servo8_offset">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="servo6_slider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>105</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="servo6_offset">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo 10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="servo10_offset">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo 4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>230</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo 8</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="servo4_offset">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QPushButton" name="save_btn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>servo1_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>servo2_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_23</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>servo9_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_9</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>servo4_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_18</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>711</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>servo3_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_3</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>servo10_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_21</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>710</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>servo11_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_12</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>servo5_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_5</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>servo7_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>servo8_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_20</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>servo12_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_17</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>718</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>servo6_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_14</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>692</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>714</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
